src/apps/miscapps/h3ToGeoBoundaryHier.c

```diff
@@ -27,21 +28,23 @@
  *  `resolution` should be a positive integer. The default is 0 (i.e., only the
  *       specified cell H3Index would be processed).
  *
- *  `outputMode` indicates the type of output; currently the choices are 0 for
- *       plain text output (the default) and 1 for KML output.
+ *  `--kml` indicates KML output format; if not specified plain text output is
```

Comment:
Again, this contains more detail than the actual help comment. We should think about how to expose this as documentation (maybe MAN pages? I have no idea how that works).